src/bcc_sensor.c

```diff
@@ -2,6 +2,22 @@
  * Copyright 2019-2020 VMware, Inc.
  * SPDX-License-Identifier: GPL-2.0
  */
+
+// Struct randomization causes issues on 4.13 and some versions of 4.14
+// These are redefined to work around this, per:
+// https://lists.iovisor.org/g/iovisor-dev/topic/21386300#1239
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 15, 0)
```

Comment:
What about kernel 4.12 and lower? Does the change hold good?

Reply:
Yeah I've also tested this on 4.8 and 4.10 and it seems fine.  From the MRs I've searched through, I believe struct randomization was actually first introduced in 4.13.